src/dotnet-interactive/CommandLine/CommandLineParser.cs

```diff
@@ -354,10 +359,10 @@ Command StdIO()
                         var vscodeSetup = new VSCodeClientKernelExtension();
                         await vscodeSetup.OnLoadAsync(kernel);
 
-                        if (startupOptions.Preview)
-                        {
-                            var http = new HttpRequestKernelExtension();
-                            await http.OnLoadAsync(kernel);
+                        if (startupOptions.Preview)
```

Comment:
Question unrelated to this PR: How are people supposed to enable this? Why not publish as an extension and let people try it out via a NuGet package?

Reply:
right now this is enabled if you use vscode insiders or via `--preview` option on the command line